rfcs/041-item-statuses/README.md

```diff
@@ -0,0 +1,311 @@
+  </tr>
+  <tr>
+    <td><code>b</code> / <code>c</code></td>
+    <td>As above</td>
+    <td>41,667</td>
+    <td>
+      Actively unhelpful.
+      This causes issues in the Catalogue API and is a custom status, so we should consider working with Collections Information to remove this.
+    </td>
+  </tr>
+  <tr>
+    <td><code>h</code></td>
+    <td>Closed</td>
+    <td>10,262</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>y</code></td>
+    <td>Permission required</td>
+    <td>8,476</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>r</code></td>
+    <td>Unavailable</td>
+    <td>5,884</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>6</code></td>
+    <td>Restricted</td>
+    <td>5,188</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>m</code></td>
+    <td>Missing</td>
+    <td>3,689</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>w</code></td>
+    <td>Dept material</td>
+    <td>151</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>p</code></td>
+    <td>In cataloguing</td>
+    <td>47</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>!</code></td>
+    <td>On holdshelf</td>
+    <td>33</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>e</code></td>
+    <td>On exhibition</td>
+    <td>24</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>s</code></td>
+    <td>On search</td>
+    <td>16</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>z</code></td>
+    <td>Claims returned</td>
+    <td>10</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>x</code></td>
+    <td>Withdrawn</td>
+    <td>10</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>a</code></td>
+    <td>a</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>d</code></td>
+    <td>On display</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>r</code></td>
+    <td>Restricted</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>t</code></td>
+    <td>In quarantine</td>
+    <td>2</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>f</code></td>
+    <td>Returned to vendor</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>0</code></td>
+    <td>0</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>q</code></td>
+    <td>Test record</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+</table>
+
+
+
+## Proposal
+
+We will not expose the Sierra item statuses.
+Instead, we will create a new set of statuses, which will be mapped as follows:
+
+<table>
+  <tr>
+    <th>id</th>
+    <th>label</th>
+    <th>Sierra statuses</th>
+    <th>comment</th>
+  </tr>
+  <tr>
+    <td>available</td>
+    <td>Available</td>
+    <td><code>-</code> / Available</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>closed</td>
+    <td>Closed</td>
+    <td>
+      <code>h</code> / Closed <br/>
+    </td>
+    <td>
+      We distinguish between closed/restricted/unavailable in the access statuses.
+      I'm assuming it's a useful distinction here, but we could collapse this into "Unavailable" if not.
```

Comment:
Having the distinction feels like the right thing to do, especially when considering the different access conditions that be applied to each one.

Reply:
This is an interesting thing to look into further. I know not everything has access conditions, whereas presumably all items have a status. But what is the real difference between them, when from this list there looks to be a considerable overlap.

It could be confusing to display what looks like the same info twice, if it's not clear that one is about usage and the other is about whether you can request the thing.

Ie, do we really use item status as a way to determine whether the thing can be requested, rather than displaying the values directly?